src/browser/ui/__tests__/ReactDOMComponent-test.js

```diff
@@ -775,4 +775,39 @@ describe('ReactDOMComponent', function() {
     });
   });
 
+  describe('option update', function(){
+    var container = document.createElement('div');
+    var variable = 10;
+    React.render(<select><option>{variable}$</option></select>, container);
+
+    it('support number update',function(){
```

Comment:
nit: **supports** number update, so that "it supports number update" reads correctly (same for all of these).

also, please add a space between `,` and `function`